### PR TITLE
docs: Fix reference to shoot operations documentation

### PR DESCRIPTION
**Pull request by marc1404**
**How to categorize this PR?**

/area documentation
/kind cleanup
/platform equinix-metal

**What this PR does / why we need it**:

The file has been moved with this PR: https://github.com/gardener/gardener/pull/10436

**Which issue(s) this PR fixes**:

_n.a._

**Special notes for your reviewer**:

_n.a._

**Release note**:

```doc developer
Fixed a reference to the shoot operations documentation.
```

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -486,11 +486,11 @@ visible Git diff).
...
+If shoot creation operations fail, you can restart them. See [here](https://github.com/gardener/gardener/blob/master/docs/usage/shoot-operations/shoot_operations.md). For example, to retry a failed shoot creation:
 
 ```
 $ kubectl -n garden-em annotate shoot em-test gardener.cloud/operation=retry
 shoot.core.gardener.cloud/em-test annotated
 ```
 
-Remember that the shoot is in the _garden_ cluster, not the _soil_ cluster, so use the appropriate kubeconfig.
\ No newline at end of file
+Remember that the shoot is in the _garden_ cluster, not the _soil_ cluster, so use the appropriate kubeconfig.
```
Comment (marc1404) [inline CONTRIBUTING.md:496]: Trailing newline added

**Review by RiRa12621 — APPROVED** [review]
/lgtm

**Pull request merged**